apps/server/src/modules/files-storage/entity/filerecord.entity.ts

```diff
@@ -145,11 +153,18 @@ export class FileRecord extends BaseEntityWithTimestamps {
 		this._creatorId = userId !== undefined ? new ObjectId(userId) : undefined;
 	}
 
-	@Property({ fieldName: 'school' })
-	_schoolId: ObjectId;
+	@Property({ fieldName: 'storageLocationId' })
+	_storageLocationId: ObjectId;
```

Comment:
```suggestion
	@Property()
	storageLocationId: ObjectId;
```

Reply:
we implemented it the same way as `_parentId `and `_creatorId`, which have a getter to return the ObjectId as String, so I would let `_storageLocationId` remain as it is and only change `stroageLocation`.

Reply:
Yes, getStorageLocationId(): string 😉